### PR TITLE
Add Smudgeplot and Cactus

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR includes two tools wrapped by the Australian people: Cactus and Smudgeplot.

### CODE REVIEW

File: galaxy-australia.yaml [hunk 1]
```diff
@@ -6,3 +6,9 @@ tools:
   - name: alphafold2
     owner: galaxy-australia
     tool_panel_section_label: ChemicalToolBox
+  - name: smudgeplot
```
[the remark this answers is not in the document]
Comment (Slugger70) [reply galaxy-australia.yaml:9]: That is the plan. I will have a chat with our tool folks and see where they're at. I think they are working on the tests and getting them to pass in IUC.

[the remark this answers is not in the document]
Comment (TomHarrop) [reply galaxy-australia.yaml:9]: Hi folks (I mean g'day)

I wrote the cactus wrapper. I'd prefer to have cactus in tools-iuc but it's Singularity-only so it fails the tests (see here: https://github.com/TomHarrop/tools-iuc/pull/5).

Cactus developers are aware of the issue with the Docker container (https://github.com/ComparativeGenomicsToolkit/cactus/issues/677). I see they have released a new version, I am trying that now.

[the remark this answers is not in the document]
Comment (bgruening) [reply galaxy-australia.yaml:9]: Input or export test data?

[the remark this answers is not in the document]
Comment (bgruening) [reply galaxy-australia.yaml:9]: @TomHarrop please also note that if you rely on Containers you exclude a bunch of Galaxy server from using the tool. That is why we stil recommend conda packages. Usegalaxy.* will work :-)

[the remark this answers is not in the document]
Comment (TomHarrop) [reply galaxy-australia.yaml:9]: The input test data for the export tool is 14 MB :)

Thanks - if there was a working and up to date conda recipe I'd use it. In this case, the developer provides and supports a container with each release. Personally, I guess that using the supported container is more likely to be reliable than having me or someone else who's not the developer write and maintain a conda recipe.

Also, from their issues page I understood that it's hard to get this software working via conda, but I can't find the specific issue where it was discussed.

[the remark this answers is not in the document]
Comment (bgruening) [reply galaxy-australia.yaml:9]: > The input test data for the export tool is 14 MB :)
> 
> Thanks - if there was a working and up to date conda recipe I'd use it. In this case, the developer provides and supports a container with each release. Personally, I guess that using the supported container is more likely to be reliable than having me or someone else who's not the developer write and maintain a conda recipe.
> 
> Also, from their issues page I understood that it's hard to get this software working via conda, but I can't find the specific issue where it was discussed.

Might be, just remember its more admin work involved for such containers as for biocontainers as well and we do not have any backups for them.

**Comment by gallardoalba** [comment]
I think this PR can be closed.

**Pull request closed without merging**